src/lake/tests/lean/test.sh

```diff
@@ -6,5 +6,5 @@ LAKE=${LAKE:-../../.lake/build/bin/lake}
 ./clean.sh
 
 $LAKE lean Test.lean -v | grep --color "Hello from the library foo!"
-$LAKE lean Test.lean -- --run Bob | grep --color "Hello Bob!"
+$LAKE lean Test.lean -- --run Test.lean Bob | grep --color "Hello Bob!"
```

Comment:
/cc @tydeu -- unfortunately I really didn't see a way to avoid this regression. A GitHub search did not show any other uses of `--run` after the module name.